device/rsvp/rsvpFastReroute.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    Rsvp.FastReroute:
+      description: >-
+        Configuration for the optional RSVP-TE FAST_REROUTE object included in Path Messages as defined in RFC4090. 
+      type: object
+      properties:
+        setup_priority:
+          description: >-
+            Specifies the value of the Setup Priority field. This controls whether the backup LSP should pre-empt existing
+            LSP that is setup with certain Holding Priority. While setting up a backup LSP, preemption of existing LSP can happen 
+            if resource limitation is encountered (e.g bandwidth availability).
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        holding_priority:
+          description: >-
+            Specifies the value of the Holding Priority field. This controls whether a new LSP being created with certain Setup Priority
+            should pre-empt this LSP set up with this Holding Priority. While setting up a new LSP, preemption of existing LSP can happen 
+            if resource limitation is encountered (e.g bandwidth availability).            
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        hop_limit:
+          description: >-
+            Specifies the value of the Hop Limit field. This controls the maximum number of hops the LSP should traverse to reach the 
+            LSP end-point.
+          type: integer
+          default: 3
+          minimum: 0
+          maximum: 255
+        bandwidth:
+          description: >-
+            Specifies the value of the Bandwidth field as a 32-bit IEEE floating point integer, in
+            bytes per second.desired for the LSP. 
```

Comment:
bytes per second.desired for the LSP. -->  the "period" between second and desired?